[DOCS] [7.8] Clarify HTTPS usage for create key API (#60858)
elastic/elasticsearch: x-pack/docs/en/rest-api/security/create-api-keys.asciidoc

`Authorization` header with a value having the prefix `ApiKey` followed
by the _credentials_, where _credentials_ is the base64 encoding of `id` and `api_key` joined by a colon.

> NOTE: If your node is configured to use SSL in a production environment, then you must specify `https` when creating your API key.

Reviewer: ```suggestion
NOTE: If your node has `xpack.security.http.ssl.enabled` set to `true`, then you must specify `https` when creating your API key.
```

The "production environment" wording is a bit misleading, since you can have SSL enabled in a non-production environment, in which case you would still need `https`.

It's also necessary (if we want to be perfectly accurate) to distinguish between SSL for the transport port and the HTTP port. 
I used the setting name because it read better to me that "If you use SSL on the HTTP port ..."
